### PR TITLE
fix renamed app.asar detection on Windows

### PR DESCRIPTION
**Pull request by DiamondMiner88**
guh

### CODE REVIEW

File: src/patcher.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ console.log("[Vencord] Starting up...");
 const injectorPath = require.main!.filename;
 
 // special discord_arch_electron injection method
-const asarName = injectorPath.endsWith("app.asar/index.js") ? "_app.asar" : "app.asar";
+const asarName = require.main!.path.endsWith("app.asar") ? "_app.asar" : "app.asar";
```
Comment (Vendicated) [inline src/patcher.ts:33]: out of curiosity, what is require.main!.filename on Windows?

File: README.md [hunk 2]
```diff
@@ -18,6 +18,11 @@ If you're just a normal user, use [our simple gui installer!](https://github.com
 
 If you wanna make plugins or contribute or just want to build from source and install manually, read [Megu's Installation Guide!](docs/1_INSTALLING.md)
 
+### For Canary/PTB users
```
Comment (Vendicated) [inline README.md:21]: no, the installer should do this

File: src/patcher.ts (hunk 1 above)
Comment (DiamondMiner88) [reply src/patcher.ts:33]: ~AppData\\Local\\DiscordCanary\\app-1.0.51\\resources\\app.asar\\index.js

Comment (Vendicated) [reply src/patcher.ts:33]: OH it just failed due to using \ instead of / lmao

**Pull request merged**